gulpfile.ts

```diff
@@ -241,55 +233,12 @@ export class Gulpfile {
             }));
     }
 
-    /**
-     * Runs unit-tests.
-     */
-    @Task()
-    unit() {
-        chai.should();
-        chai.use(require("sinon-chai"));
-        chai.use(require("chai-as-promised"));
-        return gulp.src("./build/compiled/test/**/*.js")
-            .pipe(mocha());
-    }
-
-    /**
-     * Runs before test coverage, required step to perform a test coverage.
-     */
-    @Task()
-    coveragePre() {
-        return gulp.src(["./build/compiled/src/**/*.js"])
-            .pipe(istanbul())
-            .pipe(istanbul.hookRequire());
-    }
-
-    /**
-     * Runs post coverage operations.
-     */
-    @Task("coveragePost")
-    coveragePost() {
-        chai.should();
-        chai.use(require("sinon-chai"));
-        chai.use(require("chai-as-promised"));
-
-        return gulp.src(["./build/compile/test/**/*.js"])
-            .pipe(mocha())
-            .pipe(istanbul.writeReports());
-    }
-
-    @Task()
-    coverageRemap() {
-        return gulp.src("./coverage/coverage-final.json")
-            .pipe(remapIstanbul())
-            .pipe(gulp.dest("./coverage"));
-    }
-
     /**
      * Compiles the code and runs tests.
      */
     @SequenceTask()
     tests() {
-        return ["clean", "compileTests", "tslint", "coveragePre", "coveragePost", "coverageRemap"];
+        return ["clean", "compileTests"];
```

Comment:
Please do not remove `tslint` task from this sequence. We should force contributors to write tslint/eslint compliance code. Also I think it should be run before tests.

Reply:
I removed tslint because it appeared to not be linting any files.